### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by volodymyr4501**
- [DEMO LINK](https://volodymyr4501.github.io/layout_catalog_hovers/)

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -14,6 +14,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```
Comment (Virass) [inline src/styles/_catalog.scss:17]: Use variable for duration

File: src/styles/_header.scss [hunk 2]
```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```
Comment (Virass) [inline src/styles/_header.scss:40]: Dublicate animation duration, use variable

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1 @@
...sheet" href="main.feda8047.css"></head><body> <header class="header"> <a class="header__logo" href=""> <img src="logo.6c938e44.png" alt="Moyo logo"> </a> <nav class="nav"> <ul class="nav__list"> <li class="nav__item nav__item--is-active"> <a href="" class="nav__link">Apple</a> </li> <li class="nav__item"> <a href="" class="nav__link">Samsung</a> </li> <li class="nav__item"> <a href="" class="nav__link">Смарфоны и телефоны</a> </li> <li class="nav__item"> <a href="" class="nav__link">Ноутбуки и компьютеры</a> </li> <li class="nav__item"> <a href="" class="nav__link">Гаджеты</a> </li> <li class="nav__item"> <a href="" class="nav__link">Планшеты</a> </li> <li class="nav__item"> <a href="" class="nav__link">Фото</a> </li> <li class="nav__item"> <a href="" class="nav__link">Видео</a> </li> </ul> </nav> </header> <main class="catalog"> <div class="catalog__card"> Product card </div> <div class="catalog__card"> Product card </div> <div class="catalog__card"> Product card </div> <div class="catalog__card"> Product card </div> <div class="catalog__card"> Product card </div> <div class="catalog__card"> Product card </div> <div class="catalog__card"> Product card </div> </main> </body></html>
```
Comment (Virass) [inline dist/index.html:1]: Why html minimized? Pls formating

**Review by Virass — CHANGES_REQUESTED** [review]
And fix this glitch

https://www.loom.com/share/ba210e10ccb84763b14af4c8cec41ce7

File: dist/index.html (hunk 3 above)
Comment (volodymyr4501) [reply dist/index.html:1]: image
вроді б не згорнутий

**Comment by volodymyr4501** [comment]
- [DEMO LINK](https://volodymyr4501.github.io/layout_catalog_hovers/)

File: src/styles/_catalog.scss [hunk 4]
```diff
@@ -14,6 +14,14 @@
...
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $transition-duration;
+    cursor: pointer;
+
+    &:hover {
+      color: #34568b;
+      width: 240px;
+      height: 360px;
+    }
```
Comment (valerii-derkach) [inline src/styles/_catalog.scss:24]: use scale property instead of hardcoding width and height

**Review by valerii-derkach — CHANGES_REQUESTED** [review]

File: src/styles/_catalog.scss (hunk 4 above)
Comment (volodymyr4501) [reply src/styles/_catalog.scss:24]: я не знаю як це робити

**Comment by volodymyr4501** [comment]
- [DEMO LINK](https://volodymyr4501.github.io/layout_catalog_hovers/)

File: src/styles/untils/blocks/_catalog.scss [hunk 5]
```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: $transition-duration;
+    cursor: pointer;
+
+    &:hover {
+      color: #34568b;
+    }
```
Comment (ArturErmolenko) [inline src/styles/untils/blocks/_catalog.scss:22]:
```suggestion    
    transition: transform $transition-duration, color $transition-duration;
     
      &:hover {
      transform: scale(1.2);
      color: #34568b;
    }
```

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]

File: src/styles/_catalog.scss (hunk 4 above)
Comment (ArturErmolenko) [reply src/styles/_catalog.scss]: https://mate.academy/learn/css-advanced/transformations-and-animations#/video/937

**Comment by volodymyr4501** [comment]
- [DEMO LINK](https://volodymyr4501.github.io/layout_catalog_hovers/)

**Comment by volodymyr4501** [comment]
maybe on this time demo link will be ok, I a bit fixed it.

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/94289038/193913146-258fb458-7ad3-43e2-a720-7a84d08b54e1.mov

this UI is buggy
code looks ok except one comment - You should specify transition properties directly
Default is all, but it could affect old browsers' performance 

ask in fe_chat and tag mentors - now looks like you code doesn't match the demo

**Comment by volodymyr4501** [comment]
- [DEMO LINK](https://volodymyr4501.github.io/layout_catalog_hovers/)

**Review by qoa11a — CHANGES_REQUESTED** [review]
I see the same problems that were mentioned in previous comments.
Make sure you've deployed your changes. Otherwise, please, fix these mistakes, redeploy site and request review again.

**Comment by volodymyr4501** [comment]
- [DEMO LINK](https://volodymyr4501.github.io/layout_catalog_hovers/)
-

**Review by qoa11a — CHANGES_REQUESTED** [review]
image
If you have errors that you can see, please ask for help in the fe_chat in Slack instead of asking it in Pull Request.

**Comment by volodymyr4501** [comment]
- [DEMO LINK](https://volodymyr4501.github.io/layout_catalog_hovers/)

**Comment by volodymyr4501** [comment]
Виправив марджин думаю тепер ок

**Comment by volodymyr4501** [comment]
> Виправив марджин, думаю тепер ок.

File: src/styles/untils/blocks/_catalog.scss [hunk 6]
```diff
@@ -2,7 +2,7 @@
   display: flex;
   flex-wrap: wrap;
 
-  max-width: 944px;
+  max-width: 744px;
```
Comment (stanislav-topikha) [inline src/styles/untils/blocks/_catalog.scss:5]: why? there should be 4 cards in a row

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]

**Comment by volodymyr4501** [comment]
I don't understand why you see 944 pixels if I made 744 pixels and I did npm deploy

**Comment by volodymyr4501** [comment]
image

**Comment by volodymyr4501** [comment]
- [DEMO LINK](https://volodymyr4501.github.io/layout_catalog_hovers/)

**Comment by volodymyr4501** [comment]
- [DEMO LINK](https://volodymyr4501.github.io/layout_catalog_hovers/)

**Review by polosanya — CHANGES_REQUESTED** [review]
Cards are not centered. Also, there should be 4 cards in a row.
Seems like you somehow changed default styles. You should only add hovers 
image

**Pull request closed without merging**